bower.json

```diff
@@ -36,8 +38,7 @@
       },
       "devDependencies": {
         "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
-        "promise-polyfill": "polymerlabs/promise-polyfill#1 - 2",
-        "test-fixture": "PolymerElements/test-fixture#^1.0.0",
+        "promise-polyfill": "PolymerLabs/promise-polyfill#1 - 2",
```

Comment:
Hmm, this shouldn't be here in `master`... 🤔

Reply:
What? Check #39 convo hope we didn't get it wrong lol

Reply:
Like it doesn't need 2.0 on the dep. Regardless, it's fine because it'll always be the 1.0 version as it directly depends on polymer 1.9